### PR TITLE
Add documentation for the Lua SDK

### PR DESCRIPTION
**Pull request by matthewthechickenman**

### CODE REVIEW

File: content/libraries/lua.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+name: Lua
+description: Official top.gg Lua library
```
Comment (jpbberry) [inline content/libraries/lua.mdx:3]: Change top.gg to Top.gg

**Review by jpbberry — CHANGES_REQUESTED** [review]

File: content/libraries/lua.mdx (hunk 1 above)
Comment (matthewthechickenman) [reply content/libraries/lua.mdx:3]: right

**Comment by Xetera** [comment]
do you guys actually know how to write idiomatic lua code or are you just doing this for the role on discord lol

**Comment by Zorotic** [comment]
> do you guys actually know how to write idiomatic lua code or are you just doing this for the role on discord lol

I don't know much lua but this seems like they're doing it to make the lib. even if they don't use lua I see this as them taking the opportunity to make the lib in case anyone uses lua and wants a lib for the top.gg api.

**Pull request closed without merging**